doc/manuscript/translocation.qmd

```diff
@@ -462,6 +463,13 @@ Specifically, the combined results from our reintroduction study and viability m
 In addition, given the repeatability of frog survival at a site, 1-year frog survival also serves as an effective proxy of site quality (i.e., the ability of a site to support high frog survival and a viable frog population over the long term).
 This proxy of site quality is important in the MYL frog system because accurately predicting the ability of a site to support a viable frog population *a priori* remains difficult, even after conducting 24 translocations over 16 years.
 
+The results of the meta-analysis indicate the important influence of several variables on adult survival, and by extension on population viability.
+Although we assume that the effects of winter severity and frog sex are direct effects of the variables themselves, the elevation and donor population variables are likely serving as proxies for a range of factors.
+For example, elevation is positively correlated with several attributes that may influence frog survival, including the presence of boulder habitat that is important for frog overwintering, abundance of frog predators including garter snakes, and frog developmental rates.
+The latter can influence the duration of frog life stages that have differential susceptibility to chytridiomycosis (see below).
+The duration of the tadpole and juvenile stages increases with elevation, but the fact that population growth rate is relatively insensitive to tadpole and juvenile vital rates suggests that developmental rates of early frog life stages have little effect on the predicted long-term viability of reintroduced populations.
```

Comment:
Could we phrase this slightly differently?  For example, "...the fact that long-run population growth rate is relatively insensitive to perturbations in the duration of the tadpole or juvenile stage suggests..."

Reply:
Thanks for the suggested wording. I've made this change and will merge into main.